dev/breeze/src/airflow_breeze/commands/sbom_commands.py

```diff
@@ -321,14 +370,14 @@ def generate_providers_requirements(
                     pool.apply_async(
                         get_requirements_for_provider,
                         kwds={
-                            "provider_id": p_id,
-                            "provider_version": p_version,
-                            "airflow_version": a_version,
-                            "python_version": python,
+                            "provider_id": provider_id,
+                            "provider_version": provider_version,
+                            "airflow_version": airflow_version,
+                            "python_version": python_version,
                             "force": force,
```

Comment:
I think it is better if you add "output" parameter to the "get_requirements__for_provider" and pass "output": output in here (and then use `output` in get_console() and run_command() inside the method (pass it down if needed)? 

See how it is done in other cases. This is the way how the parallel outputs are nicely handled - this way outputs are not printed to stdout but they are stored in the files (separate for each output). So when commands are running you just see "progress" information happening  (last line of the output).- very similar to what you see in CI when tests are running / images are build. 

Then you will only see by default full output when one of those commands fail. 

This will make it much easier to see what's going on.

Reply:
Thanks for the tips. Done.